### PR TITLE
Adding ability to remove custom branding image

### PR DESCRIPTION
**Pull request by hadicheaito1**
## Description

- Added ability to remove the custom branding image and make it return to the bbb_logo.png default
- Solves #4376 
## Testing Steps

- Add custom image
- Click on remove custom branding image button
- Make sure it goes back to default
## Screenshots (if appropriate):

image

### CODE REVIEW

**Comment by hadicheaito1** [comment]
I could keep it like this, or create a separate endpoint. Let me know what you guys think.

File: app/controllers/api/v1/admin/site_settings_controller.rb [hunk 1]
```diff
@@ -30,7 +30,13 @@ def update
           return render_error status: :not_found unless site_setting
 
           update = if params[:name] == 'BrandingImage'
-                     site_setting.image.attach params[:site_setting][:value]
+                     # Case where user is removing the custom branding image
+                     if params[:site_setting][:value].nil?
+                       site_setting.image.purge
```
Comment (farhatahmad) [inline app/controllers/api/v1/admin/site_settings_controller.rb:35]: Can you try replacing this with `site_setting.image.attach ActionController::Base.helpers.image_path('bbb_logo.png')`

Comment (hadicheaito1) [reply app/controllers/api/v1/admin/site_settings_controller.rb:35]: Yeah this is what I initially tried to do but it wasn't working due to the image_path not getting the correct path. I tried other helper methods to get the correct path, but the only alternative to what I did that worked was this:
`site_setting.image.attach(io: File.open('/home/blindsidenetworks/greenlight/app/assets/images/bbb_logo.png'), filename: 'bbb_logo.png')`, which I'm not sure is better. Let me know what you think, also let me know what you think about creating it's own endpoint for this if I keep the `purge`

Comment (farhatahmad) [reply app/controllers/api/v1/admin/site_settings_controller.rb:35]: Yeah I think creating a separate endpoint then would make more sense. The logic here is becoming a bit much

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**